Bottom: 8c93e60df596caf9b013f5939b10be8d231ebae8
Top:    51f5a04467d5de8d69cf4f4cbecb97a1a955ca14
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-07 20:35:12 +0000

Avoid clearing pins / vias from polygons... it is already done!





---

diff --git a/src/draw.c b/src/draw.c
index 8143cea..1cacc5c 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
